icarus/egg-icarus--dedicated.json

```
@@ -16,7 +16,7 @@
         "Wine Staging": "ghcr.io\/parkervcp\/yolks:wine_latest"
     },
     "file_denylist": [],
-    "startup": "wine64 .\/Icarus\/Binaries\/Win64\/IcarusServer-Win64-Shipping.exe -Log -SteamServerName=${SERVER_NAME} -PORT=\"${SERVER_PORT}\" -QueryPort=\"${QUERY_PORT}\"",
+    "startup": "wine64 .\/Icarus\/Binaries\/Win64\/IcarusServer-Win64-Shipping.exe -Log -SteamServerName={{SERVER_NAME}} -PORT=\"{{SERVER_PORT}}\" -QueryPort=\"{{QUERY_PORT}}\"",
```

Currently servers with either a ` ` or `'` in their name will fail to start
```suggestion
    "startup": "wine64 .\/Icarus\/Binaries\/Win64\/IcarusServer-Win64-Shipping.exe -Log -SteamServerName=\"{{SERVER_NAME}}\" -PORT=\"{{SERVER_PORT}}\" -QueryPort=\"{{QUERY_PORT}}\"",
```

---

I have fixed this